### PR TITLE
Code style & docblocks

### PR DESCRIPTION
**Pull request by seanmangar**
Fix code style to PSR-2 & add missing docblocks

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,14 +3,14 @@ This plugin helps you integrate your Laravel WebApp with [Auth0](https://auth0.c
...
 
-Check our docs page to get a complete guide on how to install it in an existing project or download a pre configured seedproject:
+Check our docs page to get a complete guide on how to install it in an existing project or download a pre-configured seed project:
 
 * Regular webapp: https://auth0.com/docs/quickstart/webapp/laravel/
 * Web API: https://auth0.com/docs/quickstart/backend/php-laravel/
 
 > If you find something wrong in our docs, PR are welcome in our docs repo: https://github.com/auth0/docs
 
-### Setting up a JWKs cache
```
Comment (glena) [inline README.md]: This is meant to be JWK (json web keys) and not JWT. JWK is the endpoint that returns the public key to verify the token

Comment (seanmangar) [inline README.md]: Ok will do

**Comment by seanmangar** [comment]
@glena Please check if this PR is good to go :)

**Comment by glena** [comment]
💥  awesome!

**Pull request merged**